lib/string.flow

```diff
@@ -1312,8 +1312,21 @@ stringEncodeFix(input : string) {
 	}
 }
 
-addHyphensIntoWord(input : string, maxlngth : int) {
-	if (strlen(input) > maxlngth) substring(input, 0, maxlngth - 5)+"- "+addHyphensIntoWord(substring(input, maxlngth - 5, strlen(input) - maxlngth + 5), maxlngth) else input
+addHyphensIntoWord(input : string, maxlngth : int) -> string {
+	addHyphensIntoWordInner(input, maxlngth, "- ")
+}
+
+softHyphenSymbol = fromCharCode(0x00ad);
```

Comment:
we have softBreakSymbol / softBreakAllowAfter defined in lib/text/localization.flow